copier.yml

```diff
@@ -0,0 +1,76 @@
+# questions
+project_name:
+  type: str
+  help: What is your project name?
```

Comment:
Consider adding validation if the value is required. Consider adding text to indicate that it is optional if the value is not required.

Reply:
Done